packages/models/src/classMap.js

```diff
@@ -89,6 +90,14 @@ export default class ClassMap {
     return this.root(CodeObjectType.DATABASE);
   }
 
+  setTemporaryRoot(id) {
+    this.roots = [this.codeObjectsById[id]];
```

Comment:
Consider the possibility of multiple root node filters.

Reply:
This is slightly different from the original implementation, where the package is not included in the root.
For example, if I expand this node:
![image](https://user-images.githubusercontent.com/8737782/135660345-1e698610-2d79-4b17-901c-88391c4d8a4c.png)

The name of the node changes in the resulting graph as a result of excluding the package:
![image](https://user-images.githubusercontent.com/8737782/135660452-d08a23c4-8848-492f-81e7-763838b90c4d.png)

While it's a somewhat minor cosmetic bug, my concern is that it is unexpected ("why did the name of the node change?") and may confuse the user.

Reply:
I think we are going to implement multiple root node filters with new `Filters` panel.
I assume in this PR we focus on context menu button "Set as root" which sets new root object every time it gets clicked.
(Bug with name change was fixed)